src/backend/distributed/commands/utility_hook.c

```diff
@@ -59,6 +60,8 @@ bool EnableDDLPropagation = true; /* ddl propagation is enabled */
 PropSetCmdBehavior PropagateSetCommands = PROPSETCMD_NONE; /* SET prop off */
 static bool shouldInvalidateForeignKeyGraph = false;
 static int activeAlterTables = 0;
+static int activeCreateExtension = 0;
```

Comment:
Postgres already exports `creating_extension` under `commands/extension.h`, which seems to cover both createExtension and alterExtension. Can we rely on those?